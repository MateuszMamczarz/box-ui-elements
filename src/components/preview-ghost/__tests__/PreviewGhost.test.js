import * as React from 'react';
import { shallow } from 'enzyme';

import PreviewGhost from '../PreviewGhost';

const getWrapper = () => shallow(<PreviewGhost />);

describe('components/preview-ghost/PreviewGhost.js', () => {
    test('should render correctly', () => {
        expect(getWrapper()).toMatchSnapshot();
    });
});
