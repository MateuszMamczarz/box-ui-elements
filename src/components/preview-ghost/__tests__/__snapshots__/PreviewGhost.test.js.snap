// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/preview-ghost/PreviewGhost.js should render correctly 1`] = `
<div
  className="PreviewGhost"
>
  <Ghost
    className="PreviewGhost-ghost"
    height={288}
  />
</div>
`;
